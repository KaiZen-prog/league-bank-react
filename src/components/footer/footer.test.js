import renderer from 'react-test-renderer';
import React from 'react';

import Footer from './footer';

test(`Footer render correctly`, () => {
  const tree = renderer
      .create(
          <Footer/>
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
