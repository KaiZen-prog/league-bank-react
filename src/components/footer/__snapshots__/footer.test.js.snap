// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render correctly 1`] = `
<footer
  className="footer app__footer"
>
  <div
    className="footer__container"
  >
    <section
      className="footer__address"
    >
      <a
        className="footer__logo"
        href="#"
      >
        ЛИГА Банк
      </a>
      <div
        className="footer__info"
      >
        <p>
          150015, г. Москва, ул. Московская, д. 32
        </p>
        <p>
          Генеральная лицензия Банка России №1050
        </p>
        <p>
          Ⓒ Лига Банк, 2019
        </p>
      </div>
      <nav
        className="footer__nav footer-nav"
      >
        <ul
          className="footer-nav__list"
        >
          <li
            className="footer-nav__item"
          >
            <a
              className="footer-nav__link"
              href="#"
            >
              Услуги
            </a>
          </li>
          <li
            className="footer-nav__item"
          >
            <a
              className="footer-nav__link"
              href="#"
            >
              Рассчитать кредит
            </a>
          </li>
          <li
            className="footer-nav__item"
          >
            <a
              className="footer-nav__link"
              href="#"
            >
              Контакты
            </a>
          </li>
          <li
            className="footer-nav__item"
          >
            <a
              className="footer-nav__link"
              href="#"
            >
              Задать вопрос
            </a>
          </li>
        </ul>
      </nav>
    </section>
    <section
      className="footer__contacts"
    >
      <ul
        className="footer__contacts-list"
      >
        <li
          className="footer__contacts-item"
        >
          <a
            className="footer__tel footer__tel--mobile"
            href="tel: *0904"
          >
            *0904
          </a>
          <p
            className="footer__phone-info"
          >
            Бесплатно для абонентов МТС, Билайн, Мегафон, Теле2
          </p>
        </li>
        <li
          className="footer__contacts-item footer__contacts-item--main-phone"
        >
          <a
            className="footer__tel footer__tel--main"
            href="tel: +78001112233"
          >
            8 800 111 22 33
          </a>
          <p
            className="footer__phone-info"
          >
            Бесплатный для всех городов России
          </p>
        </li>
      </ul>
      <section
        className="footer__social social"
      >
        <ul
          className="social__list"
        >
          <li
            className="social__item"
          >
            <a
              className="social__link social__link--fb"
              href="#"
            >
              <span
                className="visually-hidden"
              >
                Мы в фейсбуке
              </span>
            </a>
          </li>
          <li
            className="social__item"
          >
            <a
              className="social__link social__link--inst"
              href="#"
            >
              <span
                className="visually-hidden"
              >
                Мы в инстаграме
              </span>
            </a>
          </li>
          <li
            className="social__item"
          >
            <a
              className="social__link social__link--twitter"
              href="#"
            >
              <span
                className="visually-hidden"
              >
                Мы в твиттере
              </span>
            </a>
          </li>
          <li
            className="social__item"
          >
            <a
              className="social__link social__link--youtube"
              href="#"
            >
              <span
                className="visually-hidden"
              >
                Наш канал на ютубе
              </span>
            </a>
          </li>
        </ul>
      </section>
    </section>
  </div>
</footer>
`;
